src/main/java/com/kingland/eip/ehm/LogTest/Main.java

```diff
@@ -0,0 +1,10 @@
+package com.kingland.eip.ehm.LogTest;
+
+public class Main {
```

Comment:
This is not a good class name, consider to change it

Reply:
Thanks.i'll solve it.